packages/enzyme-utilities/src/trigger.ts

```diff
@@ -0,0 +1,48 @@
+import {ReactWrapper, CommonWrapper} from 'enzyme';
+import {get} from 'lodash';
+
+export type AnyWrapper =
+  | ReactWrapper<any, any>
+  | ReactWrapper<any, never>
```

Comment:
Why do you need both types?

Reply:
https://github.com/Shopify/quilt/issues/61